Lib/xml/etree/ElementTree.py

```diff
@@ -885,7 +951,7 @@ def _serialize_xml(write, elem, qnames, namespaces,
                     else:
                         v = _escape_attrib(v)
                     write(" %s=\"%s\"" % (qnames[k], v))
-            if text or len(elem) or not short_empty_elements:
+            if text or len(elem) or not bool(short_empty_elements[tag]):
```

Comment:
`bool()` isn't needed inside of conditions.
```suggestion
            if text or len(elem) or not short_empty_elements[tag]:
```